apps/user-profile/src/app/components/header/header.component.html

```diff
@@ -7,6 +7,13 @@
   <a class="ml-3" [innerHTML]="logo" [routerLink]="disableLogo ? [] : ['/']" queryParamsHandling="merge"></a>
 
   <p [ngStyle]="{'color': textColor}" class="ml-3 mt-auto mb-auto">{{label}}</p>
+  <div *ngIf="activeLink && adminGuiUrl !== null" id="nav-menu-user-info" class="justify-content-end">
+    <button
+      mat-button
+      class="ml-3 mt-auto mb-auto"
+      (click)="redirectToAdminGUI()">
+      <mat-icon [ngStyle]="{'color': iconColor}">forward</mat-icon>
+      <span [ngStyle]="{'color': textColor}">Admin GUI</span>
+    </button>
```

Comment:
In the future, we will have multiple applications, where users might want to navigate (consolidator, registrar). I would implement the nav the same way you did for the admin gui.